src/test/integration/end-to-end/auth.test.js

```diff
@@ -0,0 +1,39 @@
+const assert = require('assert')
+const url = require('url')
+const faker = require('faker')
+
+const testUtil = require('../../tools/testUtil')
+const server = require('../../../app/server')
+
+afterAll(testUtil.closeDb)
+
+describe('auth', () => {
+  test('register', async () => {
+    const request = {
+      method: 'POST',
+      url: url.format({
+        protocol: 'http',
+        pathname: '/auth/register'
+      }),
+      payload: {
+        email: faker.internet.email(),
+        password: faker.internet.password()
+      }
+    }
+
+    const response = await server.inject(request)
```

Comment:
End-to-End test. Simulates a request, injects into the live server, verifies the output.